net-analyzer/hunt/hunt-1.5_p6_p1.ebuild

```diff
@@ -1,26 +1,27 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
+
 inherit flag-o-matic toolchain-funcs
 
 DESCRIPTION="tool for checking well known weaknesses in the TCP/IP protocol"
 HOMEPAGE="http://lin.fsid.cvut.cz/~kra/index.html"
 SRC_URI="
 	mirror://debian/pool/main/${PN:0:1}/${PN}/${PN}_${PV/_p*}.orig.tar.gz
-	mirror://debian/pool/main/${PN:0:1}/${PN}/${PN}_${PV/_p*}-$(ver_cut 4).$(ver_cut 6).diff.gz
-"
+	mirror://debian/pool/main/${PN:0:1}/${PN}/${PN}_${PV/_p*}-$(ver_cut 4).$(ver_cut 6).diff.gz"
+S="${WORKDIR}/${P/_p*}"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc x86"
+
 PATCHES=(
 	"${FILESDIR}"/${P/_p*}-exit.patch
 	"${FILESDIR}"/${P/_p*}-gentoo.patch
 	"${FILESDIR}"/${P/_p*}-log2.patch
 	"${FILESDIR}"/${P/_p*}-tpserv-log.patch
 )
-S=${WORKDIR}/${P/_p*}
 
 src_prepare() {
 	eapply "${WORKDIR}"/${PN}_${PV/_p*}-$(ver_cut 4).$(ver_cut 6).diff
```

Comment:
Tidy up and put in PATCHES, it’s ok to use WORKDIR in there (I avoided it for ages)